docs/tutorials/creating-your-first-substrate-chain/learn.md

```diff
@@ -0,0 +1,133 @@
+
+Blockchain networks are composed of individual **nodes** that are connected
+together on a peer-to-peer (P2P) network. Nodes are the individual computers on
+a network running the blockchain software that makes everything work.
+
+To function, a blockchain node needs:
+
+* A Database
+* P2P Networking
+* A Consensus Engine
+* Transaction Handling
+* A State Transition Function
+* and more...
+
+These technologies span a huge breadth of computer science, and thus usually
+require expert teams to develop. As a result, most blockchain projects are _not_
+developed from the ground up. Instead they are forked from already existing
+blockchains. For instance:
+
+* Bitcoin was forked to create: Litecoin, ZCash, Namecoin, Bitcoin Cash, etc...
+* Ethereum was forked to create: Quorem, POA Network, KodakCoin, Musicoin,
+  etc...
+
+![](assets/forks.png)
+
+Building blockchains in this way has serious limitations as these existing
+platform were not designed to be modified.
+
+## Substrate
+
+Substrate is an **open source**, **modular**, and **extensible** framework for
+building blockchains.
+
+Substrate has been designed from the ground up to provide a flexible framework
+for innovators to design and build their next blockchain network. It provides
+all the core components you need to build a customized blockchain node.
+
+### Substrate Node Template
+
+We provide an out-of-the-box working Substrate node in the form of the
+`substrate-node-template`, which should be compiling as you read this. Without
+making any changes, you and your friends could share this node template and
+create a working blockchain network with a cryptocurrency and everything!
+
+We will teach you how to use this node in "developer" mode, which allows you to
+run a network with a single node, and have some pre-configured user accounts
+with funds.
+
+### Substrate Runtime Module Template
+
+The Substrate framework puts an emphasis on making it easy to design custom
+block execution logic, a.k.a. the state transition function of the blockchain.
+We call this part of Substrate the **runtime**.
+
+The Substrate runtime is composed of runtime modules. You can think of these
+runtime modules as individual pieces of logic which define what your blockchain
+can do! Substrate provides you with a number of pre-built runtime modules
+collected in the **Substrate Runtime Module Library** (SRML).
+
+![Runtime Composition](assets/runtime.png)
+
+For example, the SRML includes a
+[Balances](https://substrate.dev/rustdocs/master/srml_balances/index.html)
+module that controls the underlying currency of your blockchain by managing the
+_balance_ of all the accounts in your system. 
+
+If you want to add smart contract functionality to your blockchain, you simply
+need to include the
+[Contracts](https://substrate.dev/rustdocs/master/srml_contracts/index.html)
+module.
+
+Even things like on-chain governance can be added to your blockchain by
+including modules like
```

Comment:
These modules are Polkadot specific; adding them to the runtime configuration facilitates on-chain governance based on Polkadot's governance mechanism design. Probably pedantic, but the point of this tutorial is that you can/should create custom modules for custom blockchains (instead of adding these existing modules for on-chain governance).

Reply:
... I agree, you are being pedantic here. The idea is to inform the user that modules can provide a suite of functionality, and we make it so you can access that functionality easily.